source/Sashimi.Tests.Shared/Server/TestCalamariCommandBuilder.cs

```diff
@@ -1,25 +1,41 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.IO;
 using System.Linq;
 using System.Reflection;
+using System.Text;
+using Calamari;
 using Calamari.Common;
+using Calamari.Common.Features.Processes;
+using Calamari.Common.Plumbing;
 using Calamari.Common.Plumbing.Extensions;
 using Calamari.Common.Plumbing.FileSystem;
 using Calamari.Common.Plumbing.Logging;
+using Calamari.Common.Plumbing.Variables;
+using Calamari.Tests.Helpers;
 using Calamari.Tests.Shared;
 using Calamari.Tests.Shared.Helpers;
 using Calamari.Tests.Shared.LogParser;
-using Sashimi.Server.Contracts;
 using Sashimi.Server.Contracts.ActionHandlers;
 using Sashimi.Server.Contracts.Calamari;
 using Sashimi.Server.Contracts.CommandBuilders;
 using Sashimi.Server.Contracts.DeploymentTools;
+using KnownVariables = Sashimi.Server.Contracts.KnownVariables;
 
 namespace Sashimi.Tests.Shared.Server
 {
     class TestCalamariCommandBuilder<TCalamariProgram> : ICalamariCommandBuilder where TCalamariProgram : CalamariFlavourProgram
     {
+        const string CalamaribinariesLocationEnvironmentVariable = "CalamariBinaries_RelativePath";
+
+        static class InProcOutProcOverride
+        {
+            public static readonly string EnvironmentVariable = "Test_Calamari_InProc_OutProc_Override";
```

Comment:
Use this environment variable to override the InProc/OutProc behaviour, otherwise it defaults to OutProc on Teamcity and InProc elsewhere